packages/ChangesReloaded-Core.package/CRBrowser.class/instance/selectNextChangeListItem.st

```diff
@@ -0,0 +1,4 @@
+actions
+selectNextChangeListItem
+
+	self changeListSelectionIndex: self changeListSelectionIndex \\\ self changeListItems size + 1
```

Comment:
I am once again asking for your squeak support: what is this fancy "\\\\\\"?

Reply:
modulo